src/insert.py

```diff
@@ -0,0 +1,42 @@
+"""Insert sort module."""
+
+
+def insert_sort(l):
+    """Sort a list of numbers by insert sort."""
+    if isinstance(l, list):
+        for i in range(len(l) - 1):
+            for j in range(i + 1, 0, -1):
+                if l[j] < l[j - 1]:
+                    tmp = l[j]
+                    l[j] = l[j - 1]
+                    l[j - 1] = tmp
+        return l
+    else:
+        raise TypeError('Input type must be a list.')
+
+
+if __name__ == '__main__':
+    import timeit as ti
+    list1 = [1, 2, 4, 6]
+    list2 = [9, 8, 7, 6, 5, 4, 3, 2, 1]
+    list3 = list(range(100, 0, -1))
```

Comment:
Your lists should really be the same lengths if you're trying to illustrate the relative speeds of sorting each